extensions/WEBGL_shader_pixel_local_storage/extension.xml

```diff
@@ -115,53 +124,44 @@ interface WEBGL_shader_pixel_local_storage {
       <param name="level" type="GLint"/>
       <param name="layer" type="GLint"/>
       Establishes the backing texture for the given plane.
-      <div class="note">
-        If <code>texture</code> has been deleted, generates an <code>INVALID_OPERATION</code>
-        error.
-      </div>
-      <div class="note">
-        If <code>texture</code> was generated by a different <code>WebGL2RenderingContext</code>
-        than this one, generates an <code>INVALID_OPERATION</code> error.
-      </div>
+      <p/>
+      If <code>texture</code> has been deleted, generates an <code>INVALID_OPERATION</code> error.
+      <p/>
+      If <code>texture</code> was generated by a different <code>WebGL2RenderingContext</code> than
+      this one, generates an <code>INVALID_OPERATION</code> error.
     </function>
   </newfun>
 
   <newfun>
     <function name="framebufferPixelLocalClearValuefvWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
       <param name="value" type="Float32List"/>
-      <param name="src_offset" type="GLuint" default="0"/>
+      <param name="srcOffset" type="GLuint" default="0"/>
       Sets the floating point clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>src_offset + 4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
+      <p/>
+      If <code>value</code> has less than <code>srcOffset + 4</code> elements, generates an
+      <code>INVALID_VALUE</code> error.
     </function>
   </newfun>
 
   <newfun>
     <function name="framebufferPixelLocalClearValueivWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
       <param name="value" type="Int32List"/>
-      <param name="src_offset" type="GLuint" default="0"/>
+      <param name="srcOffset" type="GLuint" default="0"/>
       Sets the integer clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>src_offset + 4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
+      <p/>
+      If <code>value</code> has less than <code>srcOffset + 4</code> elements, generates an
```

Comment:
I think a line was accidentally deleted here.

Reply:
Good catch!

Reply:
> Thanks for revising this Chris. It looks like there were a couple of small mistakes in the extension.xml. Did you try running "make" in your extensions/ directory locally?

I did run `make`, and just now ran it again. All looks right now!